examples/effects/network00.wyv

```diff
@@ -0,0 +1,8 @@
+module def network00(stdout : Stdout) //: NetworkType00
```

Comment:
Module signature ascription works now, so you can turn this into a real ascription rather than a comment